BrcmPatchRAM/BlueToolFixup.cpp

```diff
@@ -64,6 +64,20 @@ static const uint8_t kVendorCheckPatched[] =
     0xEB // jmp short
 };
 
+static const uint8_t kVendorCallbackOriginal[] =
+{
+    0x81, 0xF9,     // cmp ecx
```

Comment:
There are 3 such sequences in the latest bluetoothd. Do we need to patch all?

Reply:
Oops, didn't check this after I shortened the patch, my bad.

> Do we need to patch all?

Patching the vendor callback function is sufficient to let bluetooth turn on again.

---

I took a look into the other two, seems like it doesn't change the behavior in `sub_100234FF2` since it checks if chipset is larger or equal to 19 first.

<details>
  <summary>Screenshots</summary>
<img width="557" alt="image" src="https://user-images.githubusercontent.com/45396585/139514053-289a91c5-ce9f-4e58-98bb-054bdd304a75.png">

<img width="847" alt="image" src="https://user-images.githubusercontent.com/45396585/139514077-3c8b2a2a-20c3-4d7d-b9e2-5458fd70b926.png">
</details>
The other seems to benefit from this patch... should we keep it?
<details>
  <summary>Screenshots</summary>
<img width="710" alt="image" src="https://user-images.githubusercontent.com/45396585/139514191-01f8f130-b5d5-4b31-be1c-ba456d3ad94b.png">

<img width="826" alt="image" src="https://user-images.githubusercontent.com/45396585/139514212-27d7ea08-d596-423b-b829-e08ea2412c26.png">
</details>

Reply:
Yeah, should keep it simpler. I checked it myself and it seems safe.